### PR TITLE
MMB: connect trip overview to the real data

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: app/MMB/src/scenes/tripOverview/Timeline.js [hunk 1]
```diff
@@ -1,44 +1,65 @@
...
-  +data: React.Node[],
+  +children: React.Node[],
+  +data: TimelineDataType,
 |};
 
-export default class Timeline extends React.Component<Props> {
-  renderItem = ({ item, index }: {| +item: React.Node, +index: boolean |}) => {
-    const isLast = this.props.data.length === index + 1;
-    const lastItemStyle = {};
+class Timeline extends React.Component<Props> {
```
Comment (tbergq) [inline app/MMB/src/scenes/tripOverview/Timeline.js]: Could this be a functional component instead?

File: app/MMB/src/scenes/tripOverview/TimelineArrival.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// @flow strict
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation } from '@kiwicom/mobile-localization';
+
+function TimelineArrival(props) {
```
Comment (tbergq) [inline app/MMB/src/scenes/tripOverview/TimelineArrival.js]: No flow type?

File: app/MMB/src/scenes/tripOverview/TimelineDeparture.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// @flow strict
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation } from '@kiwicom/mobile-localization';
+
+function TimelineDeparture(props) {
```
Comment (tbergq) [inline app/MMB/src/scenes/tripOverview/TimelineDeparture.js]: No flow type?

**Comment by mrtnzlml** [comment]
@tbergq Round 2 please.

**Pull request merged**